runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -563,8 +573,9 @@ String parseLevelUpWidget(WidgetInfo levelUpLevel)
 	 * and optionally uploads it to an image-hosting service.
 	 *
 	 * @param fileName    Filename to use, without file extension.
+	 * @param subDir    Sub directory where the file will be put.
```

Comment:
To line up with the javadoc above it.
```suggestion
	 * @param subDir      Sub directory where the file will be put.
```